packages/app/src/cli/services/dev/extension/server/utilities.ts

```diff
@@ -52,13 +45,27 @@ export function getExtensionPointRedirectUrl(
       rawUrl.searchParams.append('url', getExtensionUrl(extension, options))
       rawUrl.searchParams.append('target', requestedTarget)
       break
+    case 'customer-accounts':
```

Comment:
Never mind, I think it's `customer-accounts` - should it be `customer_account` to match the old surface? This would enable us to filter for a single surface `customer_account` when connecting via the Dev Server. If the surface names are different you'd have to remove the filter and do your own filtering.

Reply:
Good question.
`customer-account` is the first part of extension target. 
`customer-accounts` is the current surface name [on cli.](https://github.com/Shopify/cli/blob/b7c367c03d6bcff7d3f31be4dd614ff61c16f8d1/packages/app/src/cli/services/dev/extension/utilities.ts#L47)
`customer_accounts` is the surface name in [partner](https://github.com/Shopify/partners/blob/069549a3cdbef4b232a1c8a1a21e5c7d36a811ee/db/data/app_surface_areas.yml#L72) and [core](https://github.com/Shopify/shopify/blob/2268ce9a5ec9b3a5e34959fbf44cd923921d71a8/components/apps/app/models/app_modules/configuration/specifications/features/argo/argo_specification.rb#L10).

Do you mean change `customer-accounts` to `customer_accounts` [here](https://github.com/Shopify/cli/blob/b7c367c03d6bcff7d3f31be4dd614ff61c16f8d1/packages/app/src/cli/services/dev/extension/utilities.ts#L47)? @vividviolet